src/sage/graphs/traversals.pyx

```diff
@@ -20,12 +20,37 @@ Graph traversals
     :meth:`~maximum_cardinality_search` | Return an ordering of the vertices according a maximum cardinality search.
     :meth:`~maximum_cardinality_search_M` | Return the ordering and the edges of the triangulation produced by MCS-M.
 
+
+ALGORITHM:
+
+For :meth:`~lex_BFS` with ``algorithm="slow"``, :meth:`~lex_DFS`,
+:meth:`~lex_UP` and :meth:`~lex_DOWN` the same generic implementation is used.
+It corresponds to an implementation the generic algorithm described in
+"Algorithm 1" of [Mil2017]_.
+
+This algorithm maintains for each vertex left in the graph a lexicographic label
+corresponding to the vertices already removed. The vertex of maximal
+lexicographic label is then removed, and the lexicographic labels of its
+neighbors are updated. Depending on how the update is done, it corresponds to
+LexBFS, LexUP, LexDFS or LexDOWN: during the `i`-th iteration of the algorithm
+`n-i` (for LexBFS and LexDOWN) or `i` (for LexDFS and LexUP) is appended (for
+LexBFS and LexUP) or prepended (for LexDFS and LexDOWN) to the lexicographic
+labels of all neighbors of the selected vertex that are left in the graph.
+
+The time complexity of the algorithm is `O(mn)` for ``SparseGraph`` and
+`O(max(mn, n^2))` for ``DenseGraph``, where `n` is the number of vertices
```

Comment:
`max(mn, n^2)` -> `\max\{mn, n^2\}`

Reply:
done in 474c551f5fb4c2783fa18d6204aaa8618ec35ef0